iac-extensions/arm/src/test/java/org/sonar/iac/arm/parser/ResourceDeclarationTest.java

```diff
@@ -19,14 +19,12 @@
  */
 package org.sonar.iac.arm.parser;
```

Comment:
It should be moved to `arm.tree.impl.json` (I will rever my changes in this file